### ISSUE TITLE
Add server-side geolocation for UseGalaxy geographies outside of Main UseGalaxy Sites

### ISSUE THREAD

**Issue by hujambo-dunia** [galaxyproject/galaxy-hub#2435]
@bebatut:
<img width="314" alt="Screenshot 2024-02-21 at 16 50 48" src="https://github.com/galaxyproject/galaxy-hub/assets/3672779/f25be8c4-52b2-4e7c-8f19-7b1f7faf51d2">
also what is the criteria to be in the list?
Could usegalaxy FR be there too?

_Glossary:_

1.  "Main UseGalaxy Sites" - defined as the 3 UseGalaxy.* sites (US, EU, AU) as of Feb 28, 2024

**Comment by hujambo-dunia** [comment]
As a reminder, current implementation of UseGalaxy dropdown-button is client-side "simulated geolocation" which requires no additional third-party connection or technology outside of user's browser. Please be aware that some enterprise-level (ie. real) geolocation involves third-party geolocation (or "share your current location" dialog boxes).

**Comment by bebatut** [comment]
> Glossary:
>
> 1.   "Main UseGalaxy Sites" - defined as the 3 UseGalaxy.* sites (US, EU, AU) as of Feb 28, 2024

Where is this definition? I can not find it in the Glossary: https://galaxyproject.org/get-started/glossary/
But, FR is in the list of UseGalaxy Resources here: https://galaxyproject.org/use/

**Comment by hexylena** [comment]
Yes, I also find the "main sites" a confusing distinction given https://galaxyproject.org/usegalaxy/, if we wanted to separately class the "global region operators" or something, that could make sense? But we just need to clarify what the purpose of that menu is, is it to bring you to the closest usegalaxy.* (sure, fine, "usegalaxy" membership involves some commitment of functionality) 

or is it just to drive traffic to a "world region operator", thus only including eu/au/us, in which case 'galaxy australia' should be 'asia/pacific' and 'europe' should probably be 'europe, middle east, and africa' to clarify whom that galaxy is for.

(love timezone based detection, how fun! and cookie/privacy intrusion free!)

**Comment by hujambo-dunia** [comment]
Glad to provide a little background. The existing state (in the context of the feature request only) was suggested by @hexylena at GOATS meeting on Jan 23, 2024 and verbally agreed upon by others at the meeting (US, EU, AU) who lacked a direct button to `UseGalaxy.*` on the Hub Home Page for years. If anyone wants to change the future state, feel free to assign this issue to a team member who wants to implement the change - at which point the "definition" would change.

**Comment by hexylena** [comment]
> button to `UseGalaxy.*`

ok, so if it is just about `UseGalaxy.*` servers, then france should be there, as it meets the requirements, and is listed on the usegalaxy page, https://galaxyproject.org/usegalaxy/ and was listed there already on Feb 28, it's been there since a few days before christmas.

**Comment by hexylena** [comment]
(Just don't want our server admins who worked hard to meet dot-star requirements to feel left out, if it's about dot-star. If the button definition currently is not dot-star we should clarify that, is all.)

**Comment by bebatut** [comment]
Yes, I see it! Thanks a lot. I close the issue
